doc/source/usersguide/installation.rst

```diff
@@ -50,24 +50,27 @@ Prerequisites
 First, make sure that you have
 
 * Python_ >= 3.6 (PyTables-3.5 was the last release with Python 2.7 support)
-* HDF5_ >= 1.8.4 (>=1.8.15 is strongly recommended)
+* HDF5_ >= 1.10.5
 * NumPy_ >= 1.19.0
 * Numexpr_ >= 2.6.2
 * Cython_ >= 0.29.21
-* c-blosc_ >= 1.4.1 (sources are bundled with PyTables sources but the user can
+* c-blosc_ >= 1.11.1 (sources are bundled with PyTables sources but the user can
   use an external version of sources using the :envvar:`BLOSC_DIR` environment
   variable or the `--blosc` flag of the :file:`setup.py`)
+* blosc2_ >= 0.3.1 This is the Python wheel containing *both* the C-Blosc2 libs
```

Comment:
Is this correct?
In other places I see blosc2>=0.6.1

Reply:
Oops.  Fixed.